ui/cryptomaterial/dropdown.go

```diff
@@ -228,20 +254,55 @@ func (d *DropDown) Layout(gtx C, dropPos int, reversePos bool) D {
 		iRight = dropPos
 	}
 
+	if d.OpenedLayoutInset == nil {
+		d.OpenedLayoutInset = &layout.Inset{
+			Left:  unit.Dp(float32(iLeft)),
+			Right: unit.Dp(float32(iRight)),
+		}
+	}
+
+	if d.Stack {
+		isOpen := d.isOpen
+		return layout.Stack{Alignment: alig}.Layout(gtx,
+			layout.Expanded(func(gtx layout.Context) layout.Dimensions {
+				d.isOpen = false
+				display := d.closedLayout(gtx, iLeft, iRight)
+				d.isOpen = isOpen
```

Comment:
why we need set isOpen is false set it at line 270 ?

Reply:
`d.closedLayout` handles display differently if d.isOpen is true/false so I'm enforcing a specific display for this stack  before display is generated.